### DISCUSSION TITLE
Bug report 2022-11-08 nightly: IOB display vanishing / re-appearing

### DISCUSSION THREAD

**Discussion by emp-00** [NightscoutFoundation/xDrip#2475] [General]
Dear Team,

Yesterday, I updated from a ~4 months-old nightly to the **2022-11-08** revision. Ever since my IoB display is "broken".

Symptom: When opening xDrip+ in the first 1-2 minutes the IoB display is vanished and only appears after this time-span. This is always the case and clearly linked to the update. Obviously a bug from my point-of-view. This is quite annoying, because I need to have a quick look at my IoB and now I need to wait until it's appearing again. See screenshot for what's happening. Thanks for looking into this.

![image](https://user-images.githubusercontent.com/60714349/202531718-4fd17cdc-5dcf-4dbf-8e7a-4ca4b52bdfef.png)

**Comment by Navid200** [comment]
Please have a look at this issue?
https://github.com/NightscoutFoundation/xDrip/issues/2435

Are you experiencing the same issue?

**Reply by emp-00** [reply]
Yes, looks like this is the same issue - however, I don't have a Fitbit and also no Smartwatch Sensor. Just a Samsung A52s and a BlueJay connected. Long story-short: I think you can attach my bug-report to the above issue. Thanks for looking into fixing this.

I just checked: My "last working good" was nightly 2022-06-20. Will need to revert to this one now.

**Reply by dxnxz** [reply]
I have the same problem with the latest Version :/

**Reply by Navid200** [reply]
@dxnxz Please see this: https://navid200.github.io/xDrip/docs/IoB_not_showing.html

**Reply by dxnxz** [reply]
Thanks alot!

**Comment by alpacat00** [comment]
I'm still experiencing this issue despite using the most up-to-date release (from September 2024). What could be the cause of this?

**Reply by Navid200** [reply]
@alpacat00 Post a screenshot of the status page.

**Reply by alpacat00** [reply]
@Navid200 I'll do that in 2-3 days when I apply a new sensor, because now I'm experiencing a different issue - IoB functionality is not working at all when I don't have real time BG data (I don't have it because my sensor has some issues with bt and I need to scan manually with nfc). I've just posted a question about that: #3913
